src/MapViewController.swift

```diff
@@ -17,12 +17,12 @@ import OnTheRoad
   case GeneralError, AnnotationDoesNotExist, APIKeyNotSet, RouteDoesNotExist
 }
 
-public class MapViewController: TGMapViewController, LocationManagerDelegate, TGRecognizerDelegate {
+public class MapViewController: UIViewController, LocationManagerDelegate, TGRecognizerDelegate {
 
   //Error Domains for NSError Appeasement
   public static let MapzenGeneralErrorDomain = "MapzenGeneralErrorDomain"
-
-
+  
+  var tgViewController: TGMapViewController = TGMapViewController()
```

Comment:
Even though I pointed out the usage of the similar architecture design in android sdk. I still do not know a use case of when a developer will be using an accessor to the underlying TangramMap! Will be good to know of one :D.